spec/documents-spec.js

```diff
@@ -0,0 +1,270 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+/* eslint-disable no-underscore-dangle */
+
+const app = require('../index');
+const Document = require('../server/models/document');
+const request = require('supertest')(app);
+
+describe('Document routes', () => {
+  let token;
+  let documentId;
+
+  beforeAll((done) => {
+    request
+    .post('/api/users/login')
+    .send({
+      username: 'maybesydney',
+      password: 'password3',
+    })
+    .end((err, res) => {
+      token = res.body.token;
+      done();
+    });
+  });
+
+  it('Creates new documents', (done) => {
+    request
+    .post('/api/documents')
+    .set('x-access-token', token)
+    .send({
+      title: 'Adele',
+      content: 'Rumour has it',
+    })
+    .end((err, res) => {
+      documentId = res.body.document._id;
+      expect(res.status).toBe(201);
+      expect(res.body.message).toBe('Document created successfully.');
+      expect(res.body.document).toBeDefined();
```

Comment:
Since you return the document after creating it, you should check the title or content to confirm.